### ISSUE TITLE
Support For Variables in .sym Files

### ISSUE THREAD

**Issue by garethellis0** [eerimoq/cantools#95]
When parsing a `.sym` file with the following symbol, I receive the below error:
```
[ExampleSymbol1]
ID=0F0h
Len=8
Var=ExampleVariable1 float 0,32
Var=ExampleVariable2 ExampleEnum1 32,1
Var=ExampleVariable3 unsigned 33,1
```

```
Traceback (most recent call last):
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\textparser.py", line 839, in parse
    return Grammar(self.grammar()).parse(tokens, token_tree)
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\textparser.py", line 640, in parse
    raise GrammarError(tokens.peek_max().offset)
textparser.GrammarError: Invalid syntax at offset 5046.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "src\shared\CanMsgs\generate_c_code_from_sym.py", line 52, in <module>
    generate_code_from_sym_file(sym_filename)
  File "src\shared\CanMsgs\generate_c_code_from_sym.py", line 34, in generate_code_from_sym_file
    dbase = load_file(database_name + ".sym", database_format="sym")
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\cantools\database\__init__.py", line 170, in load_file
    strict)
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\cantools\database\__init__.py", line 238, in load
    strict)
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\cantools\database\__init__.py", line 305, in load_string
    return load_can_database('sym')
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\cantools\database\__init__.py", line 287, in load_can_database
    db.add_sym_string(string)
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\cantools\database\can\database.py", line 198, in add_sym_string
    database = sym.load_string(string, self._strict)
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\cantools\database\can\formats\sym.py", line 490, in load_string
    tokens = Parser60().parse(string)
  File "C:\Users\Gareth\.virtualenvs\Consolidated-Firmware-74Z7PhuA\lib\site-packages\textparser.py", line 841, in parse
    raise ParseError(text, e.offset)
textparser.ParseError: Invalid syntax at line 372, column 1: ">>!<<Var=ExampleVariable1 float 0,32"
```

As best I understand it, variables are like signals, but are symbol-specific, where the type is also held with the variable in the signal.

Please let me know if you need any more information! Thank you!

**Comment by eerimoq** [comment]
Fixed in 32.5.1, available on PyPI.

Let me know when you find more bugs.

**Comment by garethellis0** [comment]
Thanks!
